### ISSUE TITLE
Feature: Custom devtools formatter for changesets

### ISSUE THREAD

**Issue by vlascik** [poteto/ember-changeset#637]
Changesets could use a custom devtools formatter for the proxies, for better readability, perhaps similar to https://github.com/simonihmig/tracked-redux/blob/master/packages/tracked-redux/src/-private/devtools.js ?

**Comment by mcfiredrill** [comment]
Yes , it would be really handy to be able to view changesets in the ember inspector as well.
